src/main/java/transforms/DateAsDateTimeTransform.java

```diff
@@ -3,31 +3,44 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.openlattice.shuttle.dates.JavaDateTimeHelper;
-import com.openlattice.shuttle.dates.TimeZones;
 import com.openlattice.shuttle.transformations.Transformation;
 import com.openlattice.shuttle.util.Constants;
 
+import java.util.TimeZone;
+
 public class DateAsDateTimeTransform extends Transformation<String> {
     private final String[] pattern;
+    private final TimeZone timezone;
 
     /**
      * Represents a transformation from string to datetime.
      *
-     * @param pattern: pattern of date (eg. "MM/dd/YY")
+     * @param pattern:  pattern of date (eg. "MM/dd/YY")
+     * @param timezone: name of the timezone
      */
     @JsonCreator
-    public DateAsDateTimeTransform( @JsonProperty( Constants.PATTERN ) String[] pattern ) {
+    public DateAsDateTimeTransform(
+            @JsonProperty( Constants.PATTERN ) String[] pattern,
+            @JsonProperty( Constants.TIMEZONE ) String timezone
+    ) {
         this.pattern = pattern;
+        this.timezone =
+                timezone == null ? TimeZone.getTimeZone( "America/New_York" ) : TimeZone.getTimeZone( timezone );
```

Comment:
Not necessary but something to think about -- it might be helpful to create our own timezones enum with more familiar/simple names (i.e. PST, EST, etc) that we can use to map to these TimeZone strings -- and then if they ever change or if we ever switch libraries or anything like that, we'll only need to update the mappings and not have to redo all existing integrations

Reply:
I feel like these are already standardised names (same in other languages) and I think all of [these](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) (including EST) would work.